pyartcd/pyartcd/pipelines/ocp4_konflux.py

```diff
@@ -157,6 +158,46 @@ async def build(self):
 
         LOGGER.info("All builds completed successfully")
 
+    def sync_images(self):
+        LOGGER.info('Syncing images...')
+
+        record_log = self.parse_record_log()
+        if not record_log:
+            LOGGER.error('record.log not found!')
+            return
+
+        built_images = [entry['name'] for entry in record_log['image_build_konflux'] if not int(entry['status'])]
+        failed_images = [entry['name'] for entry in record_log['image_build_konflux'] if int(entry['status'])]
+        if len(failed_images) <= 10:
+            jenkins.update_description(f'Failed images: {", ".join(failed_images)}<br/>')
+        else:
+            jenkins.update_description(f'{len(failed_images)} images failed. Check record.log for details<br/>')
+
+        if not built_images:
+            # Nothing to do, skipping build-sync
+            return
+
+        if self.version != '4.19':
+            return  # TODO to be removed
+
+        if self.assembly == 'test':
+            self.runtime.logger.warning('Skipping build-sync job for test assembly')
+            return
+
+        exclude_arches = ['aarch64', 'ppc64le', 's390x', 'x86_64']
```

Comment:
i think this can be `group.yml` -> `arches` ?

Reply:
Done with 2d2e537fc1b08e290b569860c909a6d600ed4cc8